SwedbankPaySDKMerchantBackend/Classes/Problems.swift

```diff
@@ -149,7 +149,7 @@ private let serverProblems = ProblemSpace(
     }
 }
 
-private func parseBadGatewayProblem(
+internal func parseBadGatewayProblem(
```

Comment:
Are all of these changes from `private` to `internal` to facilitate testing? Are all of them required?

Reply:
Yes, all to facilitate testing. Not strictly required, but to allow testing for future use-cases that are not currently known.